notifications/http_test.go

```diff
@@ -51,7 +52,7 @@ func TestHTTPSink(t *testing.T) {
 		}
 
 		// Let caller choose the status
-		status, err := strconv.Atoi(r.FormValue("status"))
+		status, err := strconv.Atoi(r.URL.Query().Get("status"))
```

Comment:
Why is this change made? `r.FormValue` is equivalent, no?

Reply:
This was failing, looks like due to a mistake I had made during the conversion, so I've set it back now.